java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/OutgoingAsync.java

```diff
@@ -97,13 +99,13 @@ public T waitForResponseOrUserEx() throws UserException {
             return get();
         } catch (InterruptedException ex) {
             throw new OperationInterruptedException(ex);
-        } catch (java.util.concurrent.ExecutionException ee) {
+        } catch (ExecutionException ee) {
             try {
                 throw ee.getCause().fillInStackTrace();
             } catch (RuntimeException ex) // Includes LocalException
-            {
-                throw ex;
-            } catch (UserException ex) {
+                {
+                    throw ex;
+                } catch (UserException ex) {
```

Comment:
It added some indentation here which looks bogus to me.
Maybe the comment at the end of this line is tripping it up?

Maybe it you replaced the `catch` line with this, it will not add this suspicious indentation:
```diff
-} catch (RuntimeException ex) // Includes LocalException
+} catch (RuntimeException ex /* Includes LocalException */)
```